pkg/common/config/config.go

```diff
@@ -586,6 +586,15 @@ func validateGCConfig(ctx context.Context, cfg *Config) error {
 		log.Error(ErrMissingTanzuKubernetesClusterUID)
 		return ErrMissingTanzuKubernetesClusterUID
 	}
+	// GCAPIVersion and GCKind parameters have been introduced for the uTKGS effort.
+	// To maintain backward compatibility with GCs created with TKC objects,
+	// we will default to the old configuration if these values are not present.
+	if cfg.GC.GCAPIVersion == "" {
+		cfg.GC.GCAPIVersion = "run.tanzu.vmware.com/v1alpha1"
```

Comment:
Also can you define these are constants in const file rather than using directly.

Reply:
Done